README.md

```diff
@@ -228,3 +228,23 @@ customElements.define('c-e', class extends HTMLElement {});
 // 'added first'
 // The document is walked to attempt upgrades.
 ```
+
+### Settings
+
+The polyfill provides a few settings to improve performance by tweaking behavior.
+These settings typically have correctness trade offs (noted below) and should be
+used with caution.
+
+* `customElements.noDocumentConstructionObserver`: Set this flag to true to
+prevent the polyfill from mutation observing and upgrading DOM as it is added.
```

Comment:
I would add "to the main document" to the end of this sentence.

Reply:
Done.